tools/wavetool.py

```diff
@@ -242,11 +288,11 @@ def parse_cmdline():
     help='sample bits of generated wave')
     parser.add_argument('-o', '--output', type=str, help='path to store generated files', default='.')
     # wave comparison arguments
-    parser.add_argument('-a', '--analyze', type=str, choices=['smart_amp', 'wov'], help='analyze recorded wave to give case verdict')
+    parser.add_argument('-a', '--analyze', type=str, choices=['smart_amp', 'wov', 'thdn', 'alsabat'], help='analyze recorded wave to give case verdict')
     parser.add_argument('-R', '--recorded_wave', type=str, help='path of recorded wave')
     parser.add_argument('-Z', '--zero_threshold', type=float, default=-50.3, help='zero threshold in dBFS')
     parser.add_argument('-H', '--hb_time', type=float, default=2.1, help='history buffer size')
-    parser.add_argument('-T', '--threshold', type=float, default=-72.0, help='expected threshold')
+    parser.add_argument('-T', '--threshold', type=float, default=-65.0, help='expected threshold')
```

Comment:
Any comment about this default value change?

Reply:
The first version I made it -72dB, but this threshold is high for our use case. eg, wov, so I have to lower the default value.